implementations/app-typescript/src/__tests__/e2e.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { Output } from "./wrap";
+import {Output, WrapInfo} from "./wrap";
 import { loadTestCases, TestCase } from "./cases";
 import { orderOutput } from "./output";
```

Comment:
Just a nitpick, We should use some tool to keep styling consistent.

Reply:
Maybe eslint?

Reply:
This sounds like a good idea for a future PR. I'll create an issue before we merge this.

Reply:
See https://github.com/polywrap/wrap-abi-bindgen/issues/6